datafusion/sqllogictest/test_files/string/string_query.slt.part

```diff
@@ -903,20 +991,25 @@ xxxxxxxxxxxXiangpeng NULL 🔥🔥🔥🔥🔥🔥datafusion数据融合 NULL
 xxxxxxxxxxxxxRaphael NULL datafusionДатаФусион NULL
 xxxxxxxxxunder_score NULL 🔥🔥🔥🔥🔥🔥🔥🔥🔥un iść core NULL
 xxxxxxxxxxxxxpercent NULL pan Tadeusz ma iść w NULL
+xxxxxxxxxxxxxxxxxxxx NULL 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥 NULL
+NULL NULL NULL NULL
 NULL NULL NULL NULL
 
-query TT
+query TTT
 SELECT
   LPAD(ascii_1, 20),
-  LPAD(unicode_1, 20)
+  LPAD(unicode_1, 20),
+  '|'
```

Comment:
Added this to diagnose failure (https://github.com/apache/datafusion/pull/13197#issuecomment-2449343727), but the failure was without this too.
it's better to keep this though. Editors like removing empty trailing whitespace.